targon/cli/get/containers.go

```diff
@@ -15,20 +15,22 @@ import (
 )
 
 var (
-	containersIpFlag string
+	containersIpFlag     string
+	containersHotkeyFlag string
 )
 
 func init() {
 	getCmd.AddCommand(containersCMD)
 	containersCMD.Flags().StringVar(&containersIpFlag, "ip", "", "IP address of the vm")
+	containersCMD.Flags().StringVar(&containersHotkeyFlag, "hotkey", "", "Miner hotkey")
```

Comment:
this can be inferred from viper config, not a flag. the hotkey is always the validator hotkey. This isnt a miner, its the validator running inside the vm